scripts/visualize-pipeline.sh

```
@@ -0,0 +1,584 @@
+    if [[ "$input" =~ ^[0-9]+$ ]]; then
+        local url="${GCS_BASE}/${DEFAULT_REPO}/${DEFAULT_PR}/${DEFAULT_JOB}/${input}/artifacts/junit_operator.xml"
+        echo "Downloading from: $url" >&2
+        curl -sf "$url" -o "$tmp_xml" || { echo "Error: Failed to download JUnit XML from build $input" >&2; exit 1; }
+        echo "$tmp_xml"
+        return
+    fi
+
+    # Prow URL — extract GCS path
+    if [[ "$input" == *"prow.ci.openshift.org"* ]]; then
+        local gcs_path
+        gcs_path=$(echo "$input" | sed 's|.*view/gs/|gs://|')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)